.github/workflows/gradle-release.yml

```
@@ -0,0 +1,52 @@
+name: Gradle Release
+
+#on:
+#  workflow_dispatch:
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: gradle
+
+    - run: ./gradlew clean app:assembleRelease
+
+    - name: Extract Version Code
+      run: |
+        echo "VERSION_CODE=$(grep -Eo 'versionCode\s+[0-9]+' app/build.gradle | grep -Eo '[0-9]+')" >> $GITHUB_ENV
+        echo "APP_ID=$(awk '/applicationId/ {gsub(/"/, "", $2); print $2; exit}' app/build.gradle)" >> $GITHUB_ENV
+        echo "Version code is: $VERSION_CODE"
+        echo "VERSION_NAME=$(awk '/versionName/ { gsub(/'\''/, "", $2); gsub(/-SNAPSHOT/, "", $2); print $2; exit }' app/build.gradle)" >> $GITHUB_ENV
+        echo "Version name is: $VERSION_NAME"
+
+    - run: git config user.name 'Nya Ξlimu'
+    - run: git config user.email 'info@elimu.ai'
+
+    - name: Create new Git tag
+      run: |
+        git tag v${{ env.VERSION_NAME }}
```

@tuancoltech We are not using `v` in front of tags in any of our other repos, so I suggest we remove it to keep things consistent across the whole collection of apps:
```suggestion
        git tag ${{ env.VERSION_NAME }}
```

---

@jo-elimu Removed.